src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -72,6 +75,13 @@ def load_arguments(self, _):
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
         c.argument('sku', help='The sku of App Configuration', arg_type=get_enum_type(['Free', 'Standard']))
+        c.argument('endpoint', help='If auth mode is "login", provide endpoint of the App Configuration in the format "https://storename.azconfig.io". You can configure the default endpoint using `az configure --defaults appconfig_endpoint=<endpoint>`', configured_default='appconfig_endpoint')
```

Comment:
>https://storename.azconfig.io [](start = 119, length = 29)

This may not be true especially when we have the georeplciation. We can just say "provide the endpoint" or  "provide the endpoint which can be retrived by 'appconfig show' command".